src/ec/packages/ec-component-link/link.test.js

```diff
@@ -43,6 +43,134 @@ describe('EC - Link', () => {
     });
   });
 
+  describe('Standalone with missing icon name', () => {
+    const options = merge(defaultDataStructure, {
+      link: {
+        type: 'standalone',
+        label: 'Standalone link with icon',
+        icon_position: 'after',
+      },
+      icon: [
```

Comment:
If I'm not wrong, you could just pass the icon object instead of wrapping it in an array

Reply:
Yep, sure ;)